CDP4Dal/Session.cs

```diff
@@ -295,12 +301,12 @@ public IEnumerable<ReferenceDataLibrary> GetEngineeringModelSetupRdlChain(Engine
         /// <summary>
         /// Open the underlying <see cref="IDal"/> and update the Cache with the retrieved objects.
         /// </summary>
-        /// <param name="activeMessageBus">Specify if the <see cref="CDPMessageBus"/> is used or not to notify listeners</param>
+        /// <param name="activeMessageBus">Specify if the <see cref="ICDPMessageBus"/> is used or not to notify listeners</param>
         /// <returns>
         /// an await-able <see cref="Task"/>
         /// </returns>
         /// <remarks>
-        /// The Cache is updated with the returned objects and the <see cref="CDPMessageBus"/>
+        /// The Cache is updated with the returned objects and the <see cref="SingletonCDPMessageBus"/>

```

Comment:
should this not be CDPMessageBus instead of SingletonCDPMessageBus? (Present couple of time in the file)